akka-stream-tests/src/test/scala/akka/stream/impl/FanoutProcessorSpec.scala

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.impl
+
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.{ Keep, Sink, Source }
+import akka.stream.testkit.StreamSpec
+import akka.stream.testkit.Utils.TE
+import akka.testkit.TestProbe
+
+class FanoutProcessorSpec extends StreamSpec {
+
+  "The FanoutProcessor" must {
+
+    // #25634
+    "not leak running actors on failed upstream without subscription" in {
```

Comment:
would `assertAllStagesStopped` catch this issue? then perhaps add that also?

Reply:
It might, as the publisher sink actor is actually spawned as a child to the stream supervisor.